src/Symfony/Component/Console/Helper/SymfonyQuestionHelper.php

```diff
@@ -38,7 +39,7 @@ public function ask(InputInterface $input, OutputInterface $output, Question $qu
 
             // make required
             if (!is_array($value) && !is_bool($value) && 0 === strlen($value)) {
-                throw new \Exception('A value is required.');
+                throw new LogicException('A value is required.');
```

Comment:
Here I've replaced the only instance of basic ` \Exception` by a more precise `LogicException`.